README.md

```diff
@@ -184,7 +184,9 @@ Our issue tracker is at <https://github.com/sympy/sympy/issues>. Please
 report any bugs that you find. Or, even better, fork the repository on
 GitHub and create a pull request. We welcome all changes, big or small,
 and we will help you make the pull request if you are new to git (just
-ask on our mailing list or Gitter).
+ask on our [mailing list](https://groups.google.com/g/sympy?pli=1) or [Gitter Channel]
+(https://gitter.im/sympy/sympy). If you further have any queries, you can find answers
```

Comment:
Both the links are already there in the _README.md_ file

Reply:
Saw just now. Thanks, but the stackoverflow part is not. I have added a link there too. 
Let me make changes and raise a PR again.